homeassistant/components/esphome/manager.py

```diff
@@ -368,15 +381,31 @@ async def on_connect(self) -> None:
 
     def _async_on_log(self, msg: SubscribeLogsResponse) -> None:
         """Handle a log message from the API."""
-        logger_level = LOG_LEVEL_TO_LOGGER.get(msg.level, logging.DEBUG)
-        if _LOGGER.isEnabledFor(logger_level):
```

Comment:
This check was to prevent the overhead of formatting the log message if we were not going to write it, but since we now align the remote logger level we are always going to log it so it was now more overhead to check it twice